src/main/resources/application.yaml

```diff
@@ -15,6 +15,7 @@ http-client:
 external-url: http://localhost:8000
 default-timeout-ms: 250
 auction:
+  update-rates-period: 900000
```

Comment:
This is not a big deal but it seems sensible to add new parameters to the end of list